plugins/outputs/sematext/sender/sender.go

```diff
@@ -0,0 +1,112 @@
+package sender
```

Comment:
This whole sender package is Sematext specific and should be moved to an own library living outside of Telegraf.